java/ql/src/Likely Bugs/Nullness/NullMaybe.ql

```diff
@@ -21,6 +21,8 @@ from VarAccess access, SsaSourceVariable var, string msg, Expr reason
 where
   nullDeref(var, access, msg, reason) and
   // Exclude definite nulls here, as these are covered by `NullAlways.ql`.
-  not alwaysNullDeref(var, access)
+  not alwaysNullDeref(var, access) and
+  // Exclude operands of `!!` in Kotlin, as `!!` means explicit opt out by the user.
+  not access.getParent() instanceof NotNullExpr
```

Comment:
Another option would be to exclude all kotlin source code from this check.

Reply:
That seems sensible to me, since the language is enforcing proper checks for us.